include/dlr_treelite.h

```diff
@@ -0,0 +1,61 @@
+#ifndef DLR_TREELITE_H_
+#define DLR_TREELITE_H_
+
+#include <treelite/c_api_runtime.h>
+#include "dlr_common.h"
+
+
+namespace dlr {
+
+/*! /brief Structure to hold Treelite Input.
+ */
+struct TreeliteInput {
+  std::vector<float> data;
+  std::vector<uint32_t> col_ind;
+  std::vector<size_t> row_ptr;
+  size_t num_row;
+  size_t num_col;
+  CSRBatchHandle handle;
+};
+
+/*! /brief Get the paths of the Treelite model files.
+ */
+ModelPath get_treelite_paths(const std::string& dirname);
```

Comment:
Let's use CamelCase to keep consistency, `GetTreeLitePath`

Reply:
fixed